src/coreclr/jit/gentree.cpp

```diff
@@ -21583,13 +21642,17 @@ GenTree* Compiler::gtNewSimdCvtNativeNode(var_types   type,
 
                         case 32:
                         {
-                            hwIntrinsicID = NI_AVX512DQ_VL_ConvertToVector256Int64WithTruncation;
+                            hwIntrinsicID = compOpportunisticallyDependsOn(InstructionSet_AVX10v1)
```

Comment:
Missed handling `NI_AVX512DQ_ConvertToVector512Int64WithTruncation` via `NI_AVX10v1_V512_ConvertToVector512Int64WithTruncation` just above this case

Reply:
I will wait to conclude the discussion https://github.com/dotnet/runtime/pull/101938#discussion_r1607346909

Reply:
Resolved with https://github.com/dotnet/runtime/pull/101938#discussion_r1609372267